package.json

```diff
@@ -12,6 +12,7 @@
     "build": "babel src --out-dir dist --source-maps",
     "test": "babel-node jasmine.js",
     "cover": "NODE_ENV=test nyc --reporter=lcov jasmine JASMINE_CONFIG_PATH=jasmine.json",
+    "cover-windows": "set NODE_ENV=test && nyc --reporter=lcov jasmine JASMINE_CONFIG_PATH=jasmine.json",
```

Comment:
@Ardeshir81 It would be nice to have single command for unix/windows. We can use [`cross-env`](https://github.com/kentcdodds/cross-env) to set environment variables correctly.